webapp/src/main/java/com/box/l10n/mojito/entity/Branch.java

```diff
@@ -63,6 +63,13 @@ public class Branch extends SettableAuditableEntity {
   @OneToOne(mappedBy = "branch", fetch = FetchType.LAZY)
   BranchStatistic branchStatistic;
 
+  @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
This implies that each and every time we want to work with a Branch entity - we will always want to retrieve its notifiers - even if the task at hand isn't to notify anything (e.g.: listing branches per repo, asset extraction etc.).

A HQL @Query might be better in this case to limit the retrieval of this data to solely the scenario that is needed in.

Reply:
yes that was kind of my goal, in fact I didn't really want to have a table for that but I didn't want to hack the list into a single column. Do you think of a use case where this will cause performance issue?